src/wrappers/cobs.py

```diff
@@ -1,15 +1,97 @@
+import shutil
+import subprocess
+from datetime import datetime
 from os import environ
+from pathlib import Path
+from tempfile import TemporaryDirectory
 
 import cobs_index as cobs
 
-
-COBS_INDEX_PATH = environ.get('COBS_INDEX_PATH', '/data/500.cobs_compact')
+COBS_CLASSIC_INDEXES_DIR = environ.get('COBS_CLASSIC_INDEXES_DIR', '/data/classic')
+COBS_SAMPLE_DIR = environ.get('COBS_SAMPLE_DIR', '/data/samples')
+COBS_CLASSIC_FILE_EXTENSION = 'cobs_classic'
 COBS_TERM_SIZE = 31
+COBS_POSITIVE_RATE = 0.4
 
 
 class Cobs:
-    def __init__(self, index_path=COBS_INDEX_PATH):
-        self.search_instance = cobs.Search(index_path)
+    def __init__(self):
+        index_paths = []
+        for path in Path(COBS_CLASSIC_INDEXES_DIR).iterdir():
+            if path.is_dir():
+                index_paths += [p for p in path.iterdir() if p.is_file()]
+        self.search_instances = [cobs.Search(str(index_path)) for index_path in index_paths]
```

Comment:
Be careful with the index path provided for the search instances. They may become stale one there is update to it.

Reply:
hmm....we could spawn a `.lock` file in the directories to indicate a new one is being built?
Any search occurs in that period will return a "please wait" message?
Later, we could handle it more gracefully like having a wait queue
What do you think?

Reply:
The way in which the cobs index is used is that the search instances are constructed at the beginning of the first call and all the following calls use the same cached search instances. That means when the search instances are constructed, the index files are opened and each search instance has a file handle for a cobs index file.  The problem we have here is that the cobs index file could be renamed to another file or renamed from another file after we have new samples inserted. At this point, the file handle each search instance has is stale.

Reply:
THat's true. Then how about we don't cache the search instances anymore? They're only file handles, pretty light objects anyway?